packages/node/package.json

```diff
@@ -32,6 +32,11 @@
   "dependencies": {
     "@amplitude/analytics-node": "^1.3.4",
     "@amplitude/analytics-types": "^1.3.1",
-    "@amplitude/experiment-core": "^0.7.2"
+    "@amplitude/experiment-core": "^0.7.2",
+    "eventsource": "^2.0.2"
+  },
+  "devDependencies": {
+    "@types/eventsource": "^1.1.15",
+    "@types/node": "18.8.0"
```

Comment:
Need to pin a version of "@types/node" so compiling ts to js doesn't throw an error. I believe it's due to `eventsource` implementing `Event` which collided with node's `Event`.

Reply:
Is this going to be a problem for customers too? That could be really poor developer experience if so, and we may want to consider an alternative implementation.

Reply:
Fixed by not using `@types/eventsource`. `eventsource` package is pure js. Type conflicts comes in `@types/eventsource`. We declare our own type for `eventsource` package.